src/core/MeshBVH.js

```diff
@@ -759,7 +760,7 @@ export class MeshBVH {
 
 				boundsTraverseOrder: box => {
 
-					return obb.distanceToBox( box, Math.min( closestDistance, maxThreshold ) );
+					return obb.distanceToBox( box );
```

Comment:
Thanks -- looking at this again it does seem like this would be the source of an issue.